README.md

```diff
@@ -100,8 +100,8 @@ like to test your local environment you'll need to export the following variable
 #Smoke/Functional Tests
 export TEST_URL=http://localhost:4452
 export S2S_URL=http://localhost:4502
-export CCD_GW_SERVICE_NAME=ccd_gw
-export CCD_GW_SERVICE_SECRET=AAAAAAAAAAAAAAAC
+export BEFTA_S2S_CLIENT_ID=ccd_gw
+export BEFTA_S2S_CLIENT_SECRET=AAAAAAAAAAAAAAAC
 export CCD_CASEWORKER_AUTOTEST_EMAIL=someemail@blob.com
 export CCD_CASEWORKER_AUTOTEST_PASSWORD=XYZT
 ```
```

Comment:
the instructions currently in the Readme regarding func test are up to date? If yes, can we improve them? For example I don't see mentioned how to run them locally

Reply:
Sure.. Ticket created for this just now. (RDM-7471)

Reply:
👍